Trimble.Modus.Components/Controls/ListView/TextCell.xaml.cs

```diff
@@ -53,16 +52,26 @@ public string Description
     public TextCell()
     {
         InitializeComponent();
+        if (DeviceInfo.Platform != DevicePlatform.WinUI)
+        {
+            UpdateBackgroundColor();
```

Comment:
Don't you have to do this for other platforms as well?

Reply:
Yeah, the selected background color is not visible in windows when giving the background color. For windows, we are giving the colors in the App.xaml to support this we are giving the background as white in list view and not setting the background of view cell